enterprise/models.py

```diff
@@ -685,15 +685,16 @@ def notify_enrolled_learners(self, catalog_api_user, course_id, users, admin_enr
         elif self.has_multiple_idps and self.default_provider_idp:
             params = {'tpa_hint': self.default_provider_idp.provider_id}
         course_path = urlquote("{}?{}".format(course_path, urlencode(params)))
-
         lms_root_url = utils.get_configuration_value_for_site(
             self.site,
             'LMS_ROOT_URL',
             settings.LMS_ROOT_URL
         )
-        destination_url = '{site}/{login_or_register}?next={course_path}'.format(
-            site=lms_root_url,
-            login_or_register='{login_or_register}',  # We don't know the value at this time
+        enterprise_url = '{site}{course_path}'.format(
+            site=get_configuration_value(
+                'ENTERPRISE_LEARNER_PORTAL_BASE_URL',
+                settings.ENTERPRISE_LEARNER_PORTAL_BASE_URL
```

Comment:
what if they don't have the learner portal enabled?

Reply:
good point, le'ts get clarify on that case of learner poratl off. I think it's going to be the B2C link instead which is similar to https://learning.edx.org/course/course-v1:MichiganX+py4e101x+3T2020/home?utm_source=braze&utm_medium=email&utm_campaign=enrollmentconfirmation (this example is from a prior Sailthru email I got on enrollment, for reference only)

Reply:
idk about all those utm params don't think we need those for this use case. Braze will handle separately later.. maybe best to confirm with enterprise-engg or titans folks what url makes sense here... the simplest with no adornment is of the form `https://learning.edx.org/course/course-v1:MichiganX+py4e101x+3T2020/`

Reply:
SUGGEST: Add a comment on the idea that we currently only support bulk enroll for subs which means we can assume Learner portal is enabled for the enterprise, somewhere here